test/libevm/VMTestsFiller/vmBitwiseLogicOperationTestFiller.json

```diff
@@ -1501,14 +1501,15 @@
         "expect" : {
             "0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
                 "storage" : {
+		    "0x00" : "0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffd"
                 }
             }
         },
         "pre" : {
         "0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
             "balance" : "1000000000000000000000000",
             "nonce" : "0",
-            "code" : "{ [[ 0 ]] (NOT 2 )}",
+            "code" : "{ [[ 0 ]] (NOT 2) }",
```

Comment:
Please check: It might be that the semantics in LLL changed last week concerning bnot, not and iszero. So basically it could be that you have to replace each `NOT` by `ISZERO` and each `BNOT` by `NOT`. If so, we have to change the code instead of the outcome.